modules/monitoring-about-specifying-limits-and-requests-for-monitoring-components.adoc

```diff
@@ -3,22 +3,40 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: CONCEPT
-[id="about-specifying-limits-and-requests-for-monitoring-components_{context}"]
-= About specifying limits and requests for monitoring components
 
-You can configure resource limits and request settings for core platform monitoring components and for the components that monitor user-defined projects, including the following components:
+// The final solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
```

Comment:
Can you use a different phrase here? _Final solution_ has a direct connection to the Holocaust. I now it is just a comment, 

```suggestion
// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
```

Reply:
Omg really? I am so sorry, I did not know the English phrase for that :cry:  Thank you for letting me know!